x-pack/plugins/console_extensions/server/lib/spec_definitions/json/generated/logstash.put_pipeline.json

```diff
@@ -0,0 +1,11 @@
+{
+  "logstash.put_pipeline": {
```

Comment:
Should we also add some basic body autocomplete for this endpoint in this PR?

Reply:
I looked through the docs and didn't see any request parameters. Just response parameters. Would you want to just mirror the same parameters that are in the URL params?

Reply:
Actually, upon verifying in the docs, there aren't any given URL or request parameters. Do you have any suggestions? @jloleysens

Reply:
https://www.elastic.co/guide/en/elasticsearch/reference/7.x/logstash-api-put-pipeline.html